test/browser/fragments.test.js

```diff
@@ -1902,4 +1902,206 @@ describe('Fragment', () => {
 		rerender();
 		expect(scratch.textContent).to.equal('ABC');
 	});
+
+	it('should replace node in-between children', () => {
```

Comment:
TODO: Add some tests similarly to these where the component returns a Fragment, to make sure that this code properly handles components that insert Fragments *panics - hopefully it just works!*

Reply:
Done